e2e-tests/micronaut-patio-api-version/pom.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>micronaut-patio-api-version</artifactId>
```

Comment:
look at the other modules under `e2e-tests`. Name of the artifact uses parent as prefix. 
eg, rename into: `evomaster-e2e-tests-micronaut-patio-api-version`
long story... if you curious, remind me to explain at our next meeting